src/components/AddItem.js

```diff
@@ -74,15 +75,13 @@ const AddItem = () => {
     <div>
       <h1>Smart Shopping List</h1>
       <form onSubmit={handleClick}>
-        <label htmlFor="item">Item Name:</label>
-        <input
-          type="text"
+        <h2>Add new item</h2>
+        <Input
+          required
           name="item"
-          id="item"
+          placeholder="Item Name"
```

Comment:
I don't know why but I cannot see the placeholders in the input fields. Does anyone have the same problem?